### ISSUE TITLE
Images are left running when the integration tests are finished

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#2833]
I noticed that I can't run the integration tests twice, because the second one fails with this message:

```
docker.errors.APIError: 409 Client Error: Conflict ("Conflict. The container name "/db" is already in use by container 41b29a5f05b0801d5c1dfb9f31bb770e0a8e048a9f0c80459bcba54ed71e3fe1. You have to remove (or rename) that container to be able to reuse that name.")
```

I then noticed that I had four containers running:

```
[vagrant@bodhi-dev bodhi]$ sudo docker ps 
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
12004be90184        e5abcfbbaa0f        "/bin/sh -c 'gunic..."   47 minutes ago      Up 47 minutes       8080/tcp            greenwave
cf646b1bd933        11687c1dfe81        "/bin/sh -c 'gunic..."   47 minutes ago      Up 47 minutes       8080/tcp            waiverdb
62dd33cb34a1        1a69f3418598        "/usr/sbin/httpd -..."   48 minutes ago      Up 48 minutes       80/tcp              resultsdb
41b29a5f05b0        f9b577fb1ed6        "docker-entrypoint..."   48 minutes ago      Up 48 minutes       5432/tcp            db
```

All containers should be stopped before ```bodhi-ci``` exits.

**Comment by bowlofeggs** [comment]
This might be a false alarm - I did not observe this on a later job. Perhaps it was from a branch that had the problem and it was later resolved?

**Comment by abompard** [comment]
This should be fixed by PR #2837 .

**Comment by abompard** [comment]
PR #2837 is merged, closing this.
